README.rst

```diff
@@ -1,127 +1,222 @@
 SOPS: Secrets OPerationS
 ========================
-`sops` is a cli that encrypt values of yaml, json or text files using AWS KMS.
+`sops` is a secrets management tool that encrypts YAML, JSON and TEXT files
+using AWS KMS and/or PGP (via GnuPG).
 
-Usage
------
+Requirements
+------------
+First install some libraries from your package manager:
 
-Editing
-~~~~~~~
+* RHEL family::
 
-`sops` encrypted file contain the necessary KMS information to decrypt their
-content. All a user of `sops` need is valid AWS credentials and the necessary
-permissions on KMS keys.
+	sudo yum install libyaml-devel python-devel libffi-devel
 
-Given that, the only command a `sops` user need is:
+* Debian family::
+
+	sudo apt-get install libyaml-dev python-dev libffi-dev
+
+Then install the python requirements from pip::
+
+	pip install boto3 ruamel.yaml cryptography
```

Comment:
In general, I try to propose installing everything in a venv (and I create a rule in the makefile to do so)